<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NiaPy-GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/app.ico</normaloff>../icons/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBoxExperiment">
      <property name="title">
       <string>Experiment</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="settings" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelWidgetAlgorithms">
            <property name="text">
             <string>Algorithms:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetAlgorithms"/>
          </item>
          <item>
           <widget class="QLabel" name="labelWidgetBenchmarks">
            <property name="text">
             <string>Benchmarks:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetBenchmarks"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="params" native="true">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxResults">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string>Output/Debug</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditOutput">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Experiment"/>
    <addaction name="actionOpen_Experiment"/>
    <addaction name="actionSave_Experiment"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionReport_Bug"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="text">
    <string>Report Bug...</string>
   </property>
  </action>
  <action name="actionNew_Experiment">
   <property name="text">
    <string>New Experiment</string>
   </property>
  </action>
  <action name="actionOpen_Experiment">
   <property name="text">
    <string>Open Experiment</string>
   </property>
  </action>
  <action name="actionSave_Experiment">
   <property name="text">
    <string>Save Experiment</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
